Cargo.toml

```diff
@@ -2,11 +2,18 @@
 name = "bdk-cli"
 version = "0.1.0"
 edition = "2018"
-authors = ["Alekos Filini <alekos.filini@gmail.com>", "Riccardo Casatta <riccardo@casatta.it>"]
+authors = ["Alekos Filini <alekos.filini@gmail.com>", "Riccardo Casatta <riccardo@casatta.it>", "Steve Myers <steve@notmandatory.org"]
+homepage = "https://bitcoindevkit.org"
+repository = "https://github.com/bitcoindevkit/bdk-cli"
+documentation = "https://docs.rs/bdk-cli"
+description = "A CLI library and example CLI tool based on the BDK descriptor-based wallet library"
+keywords = ["bitcoin", "wallet", "descriptor", "psbt"]
+readme = "README.md"
+license-file = "LICENSE"
```

Comment:
bdk crate doesn't recognize the license this way: "non-standard"
We may want to express this way:

```license = "MIT"```

Reply:
Oh ya, that is weird, I will fix it here and make a PR to fix it in `bdk` too.